src/transformers/MultiEdgeMerger.cc

```diff
@@ -1,23 +1,82 @@
 /*
- * Copyright (c) 2022, Martin Blicha <martin.blicha@gmail.com>
+ * Copyright (c) 2022-2023, Martin Blicha <martin.blicha@gmail.com>
```

Comment:
Can you add this file and .h to the formatter pls?

Reply:
Done